package.json

```diff
@@ -4,8 +4,10 @@
   "description": "the menu component for our FEC project",
   "main": "src/server/server.js",
   "scripts": {
```

Comment:
should there be a `server-dev` script in here?  I saw it referenced in the README